src/libraries/System.Linq.Expressions/tests/Lambda/LambdaTests.cs

```diff
@@ -782,6 +782,7 @@ public void ParameterListBehavior(int parCount, string name, bool tailCall)
         }
 
         [Theory, ClassData(typeof(CompilationTypes))]
+        [ActiveIssue("https://github.com/dotnet/runtimelab/issues/155", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
What's the exception for this?

Reply:
```
  System.PlatformNotSupportedException : Operation is not supported on this platform.
     at System.Linq.Expressions.Compiler.DelegateHelpers.MakeNewCustomDelegate(Type[]) + 0x24
     at System.Linq.Expressions.Compiler.DelegateHelpers.MakeDelegateType(Type[]) + 0x6f
     at System.Linq.Expressions.Expression.Lambda(Expression, String, Boolean, IEnumerable`1) + 0x14d
     at System.Linq.Expressions.Tests.LambdaTests.AboveByteMaxArityArg(Boolean) + 0x9a
     at System.Linq.Expressions.Tests!<BaseAddress>+0x96c9cb
     at System.InvokeUtils.CallDynamicInvokeMethod(Object, IntPtr, IntPtr, IntPtr, Object, Object[], BinderBundle, Boolean, Boolean) + 0x158

```

Reply:
Given this code will throw if ref emit is not supported, it would be better to make this a ConditionalTheory conditioned on Ref.Emit than an ActiveIssue:

https://github.com/dotnet/runtime/blob/98e3fbb6887c06ef6580e4f1ddaf323d60bfa131/src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/DelegateHelpers.cs#L110-L130